tests/testthat/test-connections.R

```diff
@@ -27,21 +49,32 @@ test_that("can make a connection from list elements", {
 })
 
 test_that("can login", {
+  #Just restart the connection here to a public server..
+  conn <- catmaid_login(server='http://hildebrand16.neurodata.io/catmaid/', Cache = FALSE)
+  
```

Comment:
I would roll back all of these changes for now.

Reply:
i.e. the rest of this file.

Reply:
I have rolled back these changes.